src/App.tsx

```diff
@@ -1,7 +1,43 @@
 import React from 'react';
 
-export const App: React.FC = () => (
-  <div className="App">
-    <p className="App__message">The last pressed key is [Enter]</p>
-  </div>
-);
+interface State {
+  pressedKey: string,
+}
+
+interface Props {
+  prop?: number;
```

Comment:
The `prop` is not used in the component, consider removing it from the Props interface if it's not needed. 🧹